redbot/core/commands/commands.py

```diff
@@ -57,6 +58,49 @@ def __init__(self, *args, **kwargs):
             checks=getattr(decorated, "__requires_checks__", []),
         )
 
+    def format_help_for_context(self, ctx: "Context") -> str:
```

Comment:
Preference thing, but we could start using `from __future__ import annotations` for this.

Reply:
I've been adding it to files I touch where it doesn't require re-evaluating existing forward references for validity. Planned, but this one can wait.